wdl-lint/tests/lints/import-placements/source.wdl

```diff
@@ -0,0 +1,17 @@
+## This is a test of import placements.
+
+version 1.1
+
+import "good"
+
+import "good"
+
+import "good"
```

Comment:
Not everyone on the team agrees with me on this (specifically I think Clay registered a disagreement), but IMO there shouldn't be blank lines _between_ import statements. I would think that check would belong within the rule titled "ImportPlacement". Are we planning on add that check either here or as part of another rule?

Reply:
I was thinking it'd be its own rule; I can see wanting imports to be at the top of the document, but perhaps allowing whitespace between them.

I assume such a rule would allow comments too, such as:

```wdl
version 1.1

# Foo
import "good"
import "good"
# Bar
import "good"

import "above whitespace is bad"
```

Reply:
Yes that's exactly how I envisioned the rule. I think Clay's case of wanting XYZ imports separate from ABC imports could be accomplished with a comment labelling the two groups rather than whitespace. I mainly want to avoid:
```
version 1.0

import "fine"

import "too_much_WS"

import "no_excuse_for_this"

import "this_is_bad"
```

Reply:
Great, I think having a `ImportWhitespace` (or similarly named) rule would make sense as a follow-up PR. Does that work for you?

Reply:
Sure, I think it's fine to be separate rules. Could see it either way, but don't have a preference